scripts/commands/updateApiDocs.ts

```diff
@@ -71,12 +71,10 @@ const readArgs = (): Arguments => {
       default: false,
       description: "Is this a prior release?",
     })
-    .option("artifact", {
-      alias: "a",
-      type: "string",
-      demandOption: true,
-      description:
-        "The URL for the CI artifact to download. Must be from GitHub Actions.",
+    .option("skip-download", {
+      type: "boolean",
+      default: false,
+      description: "Skip the download of the artifact from Box?",
```

Comment:
```suggestion
      description: "Rather than downloading the artifact from Box, reuse what is already downloaded. This can save time, but it risks using an outdated version of the docs.",
```